src/Components/bookHub/BookCard.css

```diff
@@ -0,0 +1,42 @@
+.library-display {
+    width: 90vw;
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: flex-start;
+    font-family: Arial, Helvetica, sans-serif;
+    background-color: #282c34;
```

Comment:
Can you change the background color to white for now? The main screens will have white backgrounds until the designs are ready.